app/models/assignment.js

```diff
@@ -86,23 +86,13 @@ export default class AssignmentModel extends Model {
     return this.project.get('milestones').filter(milestone => this.milestoneConstants.milestoneListByTabLookup[this.tab].includes(milestone.dcpMilestone));
   }
 
-  // The following <tab>MilestoneActual<Start/End>
-  // date fields represent different dates from different milestones
-  // depending on `tab`:
-
-  // If `tab` is `upcoming`...
-  // if the project is in public review, this field is used to display the participant's review planned start date.
-  // else this field returns null
-  @computed('tab', 'dcpLupteammemberrole', 'project.{milestones,dcpPublicstatus}')
+  // This field is used to display the participant's review planned start date.
+  // If not found, returns null
+  @computed('tab', 'dcpLupteammemberrole', 'project.milestones')
   get upcomingMilestonePlannedStartDate() {
-    if (this.tab === 'upcoming') {
-      if (this.project.dcpPublicstatus !== 'Filed') {
-        const participantMilestoneId = MILESTONE_ID_LOOKUP[this.dcpLupteammemberrole];
-        const participantReviewMilestone = this.project.get('milestones').find(milestone => milestone.dcpMilestone === participantMilestoneId);
-        return participantReviewMilestone ? participantReviewMilestone.dcpPlannedstartdate : null;
-      }
-    }
-    return null;
+    const participantMilestoneId = MILESTONE_ID_LOOKUP[this.dcpLupteammemberrole];
+    const participantReviewMilestone = this.project.get('milestones').find(milestone => milestone.dcpMilestone === participantMilestoneId);
+    return participantReviewMilestone ? participantReviewMilestone.dcpPlannedstartdate : null;
```

Comment:
I wonder if this computed could just return the result of line 94. Then it's the responsibility of the caller to pull out whatever information they want. just an idea

Reply:
Hmm, I thought about it but think for now I'll leave it to this component to pull out `dcpPlannedstartdate`, so that the template doesn't have to do an extra check for `undefined` in case the CP can't find the participant milestone.